### ISSUE TITLE
This is my test results with 1.wav,  which use your upload model.  does their any method that can i enhance results ?

### ISSUE THREAD

**Issue by Alvin2580du** [liangstein/Chinese-speech-to-text#4]
袅创纵艇婆表底禁兹禁坡跷唐那发同皮盆齐盆坏老桃奎袅疫诱映佻往库提柏库亿异养秧护难赏岗克快库畴肠冈汉二换唠恐卡鱼疫一打二孔琴异语很样环忘矿坡毫爱语滩畅汉网口槐李永伟以同逃它卡网火时场传盆话房旷集框王彭梅逃扰剧阳彭冈屠画语闹末宝毫忘啊唐逃一忘冈逃硬谊弥票意依印莺重奥样乒同很二库重表二氧运逃爬察潮笔倒订口卡已映氧养影同爬投重样事上话笔澡重忘影重逃唐同样意考笔堂重杭二王老让奥忘逃后很烧谈商秧忘盘弥涛样茶察拿毫号逃尧往雅样奥误网了已厂逃拧逃跷爬爬逃重创让号穿痛团同库意席奥伊以以同网爬啊同鱼味布顶迎图疫余基第逃坡涛魄图团他同他同他坡逃脏扭迎偶二逃抵叠姆急弹拿她急怕普它第桃禁普几去集第婴集第集尽第现集朝集二去集朝去二集一集您一去二您二去二去急去集急去集其集一集一二老二婴集二急二婴靠哇把婴婴印厅普

**Comment by liangstein** [comment]
This is not the result I have got.  There is something wrong with your code.

**Comment by hitxujian** [comment]
我也用你上传的模型，跑了一遍，5个wav结果如下（就是利用函数listen（）做预测），结果也不是很好，请问你有没有用其他技巧
一九九山年二二十的上午务四穿声看月显安人向武村碰加工嫂五人进城都体一服
看亚够考前跑后你直惊准的山却也得他王欧尧起声王的在山进回道
北积穿过云层易下一片鱼海又时头过喜过的运物一些可件然国冲绿的群山大底
王宁看被墙颠后不范云燕孙场起来及自卫不均为抓活
其书有现人原本于穷取来观邪不凑找他了莫银杷要求看四损面看富面

**Comment by liangstein** [comment]
所有的code都上传了 没有其它技巧。

**Comment by hitxujian** [comment]
谢谢分享

**Comment by pingchesu** [comment]
Sorry , if I use more file to train the model , may the result be better ? or is there any method that can enhance the result such as more mfcc features or audio data preprocessing?

**Comment by liangstein** [comment]
@pingchesu  Indeed! However the quality of the dataset is very important. mfcc features needn't to be very large, unless enough GPU power is possessed.

**Comment by pingchesu** [comment]
@liangstein thanks for answering my questions, the problem I encounterd is that some of my wav file which duration are 10 seconds , but  only contained 5 second voice. The rest of 5 seconds are noise or silent. Would you recommend some methods to solve these problems especially for speech to sentence? The way I found in the internet always talk about how to deal with speech to words ,not for speech to sentence. Sorry for bothering you!!
